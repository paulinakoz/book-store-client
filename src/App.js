

function App() {
  return (
    <div className="App">
      My first component
    </div>
  );
}

export default App;
