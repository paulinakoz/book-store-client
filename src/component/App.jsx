import React from "react";
import axios from "axios";
import baseUrl from "../config/index.js";
import Layout from "./layout/Layout.jsx";
import BookContainer from "./book/BookContainer.jsx";

function App() {
  axios(`${baseUrl}/api/v1/books`).then((books) => {
    console.log(books);
  });

  return (
    <Layout>
      <BookContainer />
    </Layout>
  );
}

export default App;
