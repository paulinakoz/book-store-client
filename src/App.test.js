import { render, screen } from '@testing-library/react';
import App from './App';

describe('App component', () => {
  it('should render without error', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  })
});
