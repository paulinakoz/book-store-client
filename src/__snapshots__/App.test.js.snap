// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render without error 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    My first component
  </div>
</DocumentFragment>
`;
